docs/sources/tempo/release-notes/v2-5.md

```diff
@@ -0,0 +1,183 @@
+* `SearchTagValuesV2`
+* `MetricsQueryRange`
+
+These new streaming endpoints are not supported by Grafana yet. However, they are supported by the Tempo CLI tool.
+
+In the Tempo CLI, you can use the `--use-grpc` option to enable GRPC streaming.
+
+To learn more, refer to the [Tempo gRPC API](https://grafana.com/docs/tempo/latest/api_docs/#tempo-grpc-api) and [Tempo CLI](https://grafana.com/docs/tempo/latest/operations/tempo_cli/#search) documentation. [PR 3460](https://github.com/grafana/tempo/pull/3460) [[PR #3584](https://github.com/grafana/tempo/pull/3584)]
+
+{{< admonition type="note" >}}
+NOTE: Streaming over HTTP requires the `stream_over_http_enabled` flag to be set. For more information, refer to [Tempo GRPC API documentation](https://grafana.com/docs/tempo/latest/api_docs/#tempo-grpc-api).
+{{< /admonition >}}
+
+In addition, we’ve reduced memory consumption in the frontend for large traces. [[PR 3522](https://github.com/grafana/tempo/pull/3522)]
+
+### New vParquet4 block format (experimental)
+
+New in Tempo 2.5, the vParquet4 block format is required for using links, events, and arrays and improves query performance relative to previous formats. vParquet4 will be the default block format in the next release. [[PR 3368](https://github.com/grafana/tempo/pull/3368)]
+
+In addition, we’ve updated the OTLP schema to add attributes to instrumentation scope in vParquet4.[[PR 3649](https://github.com/grafana/tempo/pull/3649)]
+
+While you can use vParquet4, keep in mind that it's experimental. If you choose to use vParquet4 and then opt to revert to vParquet3, any vParquet4 blocks would not be readable by vParquet3.
+
+To try vParquet4, refer to [Choose a block format](https://grafana.com/docs/tempo/latest/configuration/parquet/#choose-a-different-block-format).
+
+### TraceQL
+
+Unique to Tempo, TraceQL is a query language that lets you perform custom queries into your tracing data.
+To learn more about the TraceQL syntax, see the [TraceQL documentation](https://grafana.com/docs/tempo/latest/traceql/).
+
+For information on planned future extensions to the TraceQL language, refer to [future work](https://grafana.com/docs/tempo/latest/traceql/architecture/).
+
+We’ve made the following improvements to TraceQL:
+
+* Add support for scoped intrinsics using a colon (`:`). The available scoped intrinsics are trace:duration, trace:rootName, trace:rootService, span:duration, span:kind, span:name, span:status, span:statusMessage. [[PR [3629](https://github.com/grafana/tempo/pull/3629)]
+* Performance improvements on TraceQL and tag value search. [[PR 3650](https://github.com/grafana/tempo/pull/3650),[PR 3667](https://github.com/grafana/tempo/pull/3667)]
+* Add `nestedSetLeft/Right/Parent` intrinsics to TraceQL. [[PR 3497](https://github.com/grafana/tempo/pull/3497)]
+* Reduced memory usage and improved performance when working with large datasets with many repeated strings. [[PR 3411](https://github.com/grafana/tempo/pull/3411)]
```

Comment:
i know we wrote this together but now I'm thinking... should this match the other points and say "reduce and improve" instead of past tense?

Reply:
```suggestion
* Reduce memory usage and improve performance when working with large datasets with many repeated strings. [[PR 3411](https://github.com/grafana/tempo/pull/3411)]
```